### ISSUE TITLE
ParaglideJS component needs to be updated to Svelte 5 logics

### ISSUE THREAD

**Issue by twoends** [opral/inlang-paraglide-js#262]
Looking into ParaglideJS.svelte I noticed that the component is still svelte 4 written and I can see this error:
![image](https://github.com/user-attachments/assets/c39bae8a-5b70-4d57-827a-6fd10328d4db)

It would be nice to have it updated to svelte 5 new logics. Is it already planned?

**Comment by samuelstroschein** [comment]
if your, or someone else, makes a pr that fixes the store lint here, i'll review and merge it promptly. <br>*<br>I am busy with lix at the moment which blocks progress on inlang's side.* [https://opral.substack.com/p/focus-shift-from-inlang-to-lix](https://opral.substack.com/p/focus-shift-from-inlang-to-lix)

**Comment by add-n2x** [comment]
My browser also keeps complaining about incompatible Svelte 4 `$:`  statements:

```js
[svelte] reactive_declaration_non_reactive_propertyA `$:` 
statement (node_modules/​@inlang/​paraglide-sveltekit/​dist/​runtime/​AlternateLinks.svelte:21:0) 
read reactive state that was not visible to the compiler. 
Updates to this state will not cause the statement to re-run. 
The behaviour of this code will change if you migrate it to runes mode
```

Latest Svelte 5 and paraglide-js 1.11.3.

**Comment by samuelstroschein** [comment]
@twoends @add-n2x i only have time to review PRs atm. i put up a bounty for anyone solving this issue. i will quickly review and merge a PR

**Comment by samuelstroschein** [comment]
Released in `@inlang/paraglide-sveltekit:   0.13.0`

**Comment by samuelstroschein** [comment]
Reverted 0.13.0 due to https://github.com/opral/inlang-paraglide-js/issues/272. 

Someone needs to check what was wrong with the PR. The tests passed locally.

**Comment by samuelstroschein** [comment]
Fixed in `@inlang/paraglide-sveltekit: 0.15.0` via https://github.com/opral/monorepo/pull/3277

**Comment by fromaline** [comment]
@samuelstroschein, hey!
0.15.0 release should have been a major version. It doesn't work in Svelte 4 projects anymore

**Comment by samuelstroschein** [comment]
hey @fromaline, 

shoot. i forgot to mention it in the changelog. regarding the version number, i followed the [semver spec paragraph 4](https://semver.org/#spec-item-4):

> Major version zero (0.y.z) is for initial development. Anything MAY change at any time. The public API SHOULD NOT be considered stable.

I am hesitant to release a 1.0 of the adapter before #217.

**Comment by fromaline** [comment]
@samuelstroschein, oh, got it! I didn’t consider the pre-v1 release specifics. Other Svelte-related projects often bump the major version when dropping Svelte 4 support, so I assumed it was necessary. A note about the Svelte 4 support drop would be helpful!

By the way, do you plan to patch bugs and backport features to the `@inlang/paraglide-sveltekit` version that supports Svelte 4, or will all development now focus on the Svelte 5 compatible version? I’m asking because my upgrade to Svelte 5 is still blocked by other dependencies in my project ;(

**Comment by samuelstroschein** [comment]
No backporting is planned. The NPM download stats indicate that Svelte 4 has only a few 10-20.000 weekly downloads left compared to 300.000+ that Svelte 5 has.

**Comment by fromaline** [comment]
@samuelstroschein, got it! Thanks for being transparent, and a big shoutout for this awesome library! I love it so much compared to the alternatives — it works perfectly 99% of the time!
